docs/weekly-questions/joes-water.md

```diff
@@ -242,4 +242,33 @@ print("YES" if after_discount_applied > original_profit else "NO")
 
 </p>
 </details>
--->
+
+
+
+## Model Solution 2
+
+<details><summary>Click to reveal</summary>
+<p>
+
+Here's a more cut-down-to-the-minimum solution that may be more readable for some of you:
+
+```py
+joesWater = input().split(' ')
+competitionsWater = input().split(' ')
+discount = int(input())
+
+originalProfit = 0
+discountedProfit = 0
+
+for i in range(len(joesWater)):
+    curJoeStore = int(joesWater[i])
+    curCompStore = int(competitionsWater[i])
+    if curJoeStore < curCompStore:
+        originalProfit += curJoeStore
+    if curJoeStore - discount < curCompStore:
+        discountedProfit += curJoeStore - discount
+
+print('YES') if discountedProfit > originalProfit else print('NO')
```

Comment:
This is not valid syntax.
```suggestion
print("YES" if discountedProfit > originalProfit else "NO")
```

Reply:
It is valid, but the latter looks cleaner anyway.

Reply:
Huh, never mind, my apologies -- it is indeed valid.